### DISCUSSION TITLE
Would anyone be willing make this for me?

### DISCUSSION THREAD

**Discussion by booyakafoo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#455] [General]
I have a BestWay SaluSpa Laguna that I purchased last year.  Model shows S100105

![A22144D1-7E6C-45F1-8A25-AA2618C2AB59](https://user-images.githubusercontent.com/3913228/220478796-40117332-eaf0-4acc-bc34-166330b63888.jpeg)

I’m not familiar with PCBs, schematics, and am terrible at soldering. 

Would anyone be willing to make one for me?  I’d obviously pay for your troubles.  I live in Houston TX.  Thanks

**Comment by xxDmOxx** [comment]
I'm also looking for a 6-wire unit for sale if someone has a spare available!  Thank you!

**Comment by essi82** [comment]
I am also looking for a unit to buy for the pump s100101, am based in Finland
